### ISSUE TITLE
Update the Pin Browser 'search & destroy' method. 

### ISSUE THREAD

**Issue by ckmahoney** [warfare-plugins/social-warfare#678]
# Notes for testers
Pinterest updated their browser extension, which broke our search & destroy feature. 

1. Install or update the Pinterest extension for Chrome. Activate the extension. 
2. In Social Warfare > Display, turn Image Hover Pin Button ON and Save Changes. 
3. Set the minimum width and height to 5 so all images get a hover button. 
4. Visit a post on the frontend and hover an image in the_content. It should have a Social Warfare "Save" button rather than the Pinterest save button. 
    - If you can not tell the difference between the two, try changing the position of the SW button by changing its placement in SW > Display > Image Hover Pin Button

#
original issue

IREF 252094

In order to properly identify a browser 'Save' button, we need to know exactly what its properties are. 
Unfortunately, since the buttons are controlled by an external source, those properties are prone to change over time. 

It looks like a recent update to the Pin Browser Extension has changed the properties of the button. 

We need to update the search & destroy message to look for the new properties as well as the old.

**Comment by ckmahoney** [comment]
This issue has been annotated and is ready for testing.

**Comment by iHelpers** [comment]
The first image hover after the page loads shows both buttons. All subsequent hovers show only our button.. 
But, reload the page, and both buttons show first time, the first time, only.
![hoverzz](https://user-images.githubusercontent.com/39499655/53373028-42388500-3909-11e9-811a-bc610a8c307d.jpg)

**Comment by ckmahoney** [comment]
Yeah. The reason is because Pinterest does not actually create their button until an image is hovered the first time. 

I tried simulating the hover event on pageload hoping that it would trigger PInterest to make its button. That did not work. 

Another thought that came to mind is to use CSS to hide it. But, after inspecting their button, I found there are no selectors for it. Instead, it is a generic <span> with all of its style set inline, each with an `!important` tag.
